### PR TITLE
refactor: add open css button to theme editor

### PR DESCRIPTION
**Pull request by sissbruecker**
## Description

Adds a button for opening / editing each element's respective CSS rule in the IDE. Kept the "open CSS" terminology in the code, but thought that "Edit CSS" would be a better label for the button itself. When using local theming, the action also adds a local class name to the component if it doesn't have one yet, so that a proper selector / CSS rule can be generated.

Bildschirmfoto 2023-03-29 um 11 21 42

### CODE REVIEW

File: vaadin-dev-server/frontend/vaadin-dev-tools.test.ts [hunk 1]
```diff
@@ -12,6 +12,10 @@ describe('vaadin-dev-tools', function () {
     // @ts-ignore
     window.Vaadin = {};
     // @ts-ignore
+    window.Vaadin.Flow = {};
+    // @ts-ignore
+    window.Vaadin.Flow.clients = {};
+    // @ts-ignore
```
Comment (sissbruecker) [inline vaadin-dev-server/frontend/vaadin-dev-tools.test.ts:18]: Unrelated test fix that I missed in previous PRs, same with `package.json`

**Review by MarcinVaadin — APPROVED** [review]
Tested and working
